motion/motion.go

```diff
@@ -133,7 +137,7 @@ func (d *motionDetector) pixelsChanged(frame *lepton3.Frame) (bool, int) {
 		return false, 0
 	}
 
-	if isAffectedByFFC(frame) {
+	if isAffectedByFFC(frame) || prevFFC {
```

Comment:
What's the purpose of including `prevFFC` here? `isAffectedByFFC` returns true if it's within 10s of the last FFC. AFAICS including `prevFFC` here just extends that period by 1 frame. Is that's intended?

Reply:
Yes this sets the oldest floored frame to have an appropriate temperature threshold